docs/sources/setup-grafana/configure-docker.md

```diff
@@ -60,48 +93,262 @@ The following settings are hard-coded when launching the Grafana Docker containe
 | GF_PATHS_PLUGINS      | /var/lib/grafana/plugins  |
 | GF_PATHS_PROVISIONING | /etc/grafana/provisioning |
 
+## Install plugins in the Docker container
+
+You can install publicly available plugins and plugins that are private or used internally in an organization. For plugin installation instructions, refer to [Install plugins in the Docker container]({{< relref "./installation/docker#install-plugins-in-the-docker-container" >}}).
+
+### Install plugins from other sources
+
+To install plugins from other sources, you must define the custom URL and specify it immediately before the plugin name in the `GF_INSTALL_PLUGINS` environment variable: `GF_INSTALL_PLUGINS=<url to plugin zip>;<plugin name>`.
+
+Example:
+
+The following command runs Grafana Enterprise on **port 3000** in detached mode and installs the custom plugin, which is specified as a URL parameter in the `GF_INSTALL_PLUGINS` environment variable.
+
+```bash
+docker run -d -p 3000:3000 --name=grafana \
+  -e "GF_INSTALL_PLUGINS=http://plugin-domain.com/my-custom-plugin.zip;custom-plugin,grafana-clock-panel" \
+  grafana/grafana-enterprise
+```
+
+## Build a custom Grafana Docker image
+
+In the Grafana GitHub repository, the `packaging/docker/custom/` folder includes a `Dockerfile` that you can use to build a custom Grafana image. The `Dockerfile` accepts `GRAFANA_VERSION`, `GF_INSTALL_PLUGINS`, and `GF_INSTALL_IMAGE_RENDERER_PLUGIN` as build arguments.
+
+The `GRAFANA_VERSION` build argument must be a valid `grafana/grafana` Docker image tag. By default, Grafana builds an Alpine-based image. To build an Ubuntu-based image, append `-ubuntu` to the `GRAFANA_VERSION` build argument.
+
+Example:
+
+The following example shows you how to build and run a custom Grafana Docker image based on the latest official Ubuntu-based Grafana Docker image:
+
+```bash
+# go to the custom directory
+cd packaging/docker/custom
+
+# run the docker build command to build the image
+docker build \
+  --build-arg "GRAFANA_VERSION=latest-ubuntu" \
+  -t grafana-custom .
+
+# run the custom grafana container using docker run command
+docker run -d -p 3000:3000 --name=grafana grafana-custom
+```
+
+### Build Grafana with the Image Renderer plugin pre-installed
+
+> **Note:** This feature is experimental.
+
+Currently, the Grafana Image Renderer plugin requires dependencies which are not available in the Grafana Docker image (see [GitHub Issue#301](https://github.com/grafana/grafana-image-renderer/issues/301) for more details). However, you can create a customized Docker image utilizing the `GF_INSTALL_IMAGE_RENDERER_PLUGIN` build argument as a solution. This will install the necessary dependencies for the Grafana Image Renderer plugin to run.
```

Comment:
```suggestion
Currently, the Grafana Image Renderer plugin requires dependencies that are not available in the Grafana Docker image (see [GitHub Issue#301](https://github.com/grafana/grafana-image-renderer/issues/301) for more details). However, you can create a customized Docker image utilizing the `GF_INSTALL_IMAGE_RENDERER_PLUGIN` build argument as a solution. This will install the necessary dependencies for the Grafana Image Renderer plugin to run.
```